Build Fails on Raspberry Pi 4: No rule to make target

Vanilla build of raspbian 32-bit lite. Dependencies install and then trying to 'sudo make install' gets to 98% and then errors out:

[ 88%] Built target bif_loader_plugins
make[3]: Entering directory '/home/zeek1/downloads/zeek/build'
make[3]: Leaving directory '/home/zeek1/downloads/zeek/build'
make[3]: Entering directory '/home/zeek1/downloads/zeek/build'
make[3]: Leaving directory '/home/zeek1/downloads/zeek/build'
[ 88%] Built target bif_loader
make[3]: Entering directory '/home/zeek1/downloads/zeek/build'
make[3]: Leaving directory '/home/zeek1/downloads/zeek/build'
[ 98%] Built target zeek_objs
make[3]: Entering directory '/home/zeek1/downloads/zeek/build'
make[3]: Leaving directory '/home/zeek1/downloads/zeek/build'
make[3]: Entering directory '/home/zeek1/downloads/zeek/build'
make[3]: *** No rule to make target 'src/builtin-plugins/spicy-plugin/src/compiler/CMakeFiles/zeek-compiler.dir/driver.cc.o', needed by 'src/zeek'.  Stop.
make[3]: Leaving directory '/home/zeek1/downloads/zeek/build'
make[2]: *** [CMakeFiles/Makefile2:6910: src/CMakeFiles/zeek.dir/all] Error 2
make[2]: Leaving directory '/home/zeek1/downloads/zeek/build'
make[1]: *** [Makefile:182: all] Error 2
make[1]: Leaving directory '/home/zeek1/downloads/zeek/build'
make: *** [Makefile:15: all] Error 2

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -863,6 +863,10 @@ CheckOptionalBuildSources(auxil/zeek-archiver ZeekArchiver INSTALL_ZEEK_ARCHIVER
 CheckOptionalBuildSources(auxil/zeek-client ZeekClient INSTALL_ZEEK_CLIENT)
 
 if ( NOT DISABLE_SPICY )
+    # The `zeek` binary implicitly depends on the driver object file built
+    # as part of `spicy`; make that dependency explicit.
+    add_dependencies(zeek spicyz)
+
     if ( NOT SPICY_ROOT_DIR )
         list(APPEND _spicy_rt_libs spicy-rt hilti-rt)
 

